packages/styles/framework/src/utilities/mixins/_mixins.scss

```
@@ -155,20 +210,58 @@
+      }
+      .#{$type}-sticker {
+        top: -1px;
+        right: 15px;
+      }
+    }
+
+    &:not([aria-disabled="true"]) {
+      cursor: pointer;
+    }
+
+    &:not([aria-disabled="true"]):not([aria-checked="true"]) {
```

Stylelint (17.3.0)

[error] Expected complex :not() pseudo-class notation (selector-not-notation)

(selector-not-notation)